src/main/java/in/koreatech/koin/domain/dept/controller/DeptController.java

```diff
@@ -0,0 +1,32 @@
+package in.koreatech.koin.domain.dept.controller;
+
+import java.util.List;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import in.koreatech.koin.domain.dept.dto.DeptResponse;
+import in.koreatech.koin.domain.dept.dto.DeptsResponse;
+import in.koreatech.koin.domain.dept.service.DeptService;
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequiredArgsConstructor
+public class DeptController {
+
+    private final DeptService deptService;
+
+    @GetMapping("/dept")
+    public ResponseEntity<DeptResponse> findDept(@RequestParam(value = "dept_num") Long id) {
+        DeptResponse response = deptService.findDeptBy(id);
```

Comment:
```suggestion
    public ResponseEntity<DeptResponse> findDept(@RequestParam(value = "dept_num") Long deptNumber) {
        DeptResponse response = deptService.findDeptBy(deptNumber);
```

`id`라는 포괄적인 변수 명보다
`deptNumber` 라는 구체적인 변수 명을 활용하면 좋을 것 같습니다.